kubernetes/cray-console-operator/Chart.yaml

```diff
@@ -1,6 +1,25 @@
-apiVersion: v1
-description: "Kubernetes resources for cray-console-operator"
-name: "cray-console-operator"
-home: "scms/cray-console-operator"
-# This version string is replaced at build time by the build prep script
-version: @VERSION@
+apiVersion: v2
+name: cray-console-operator
+version: 0.0.0
+description: Kubernetes resources for cray-console-operator
+keywords:
+  - cray-console
```

Comment:
Do the other console repos have this keyword as well?

Reply:
Yes. They each have the generic `cray-console` tag, and then a more specific `cray-console-node`, `cray-console-operator` or `cray-console-data` tag.